packages/cxl-ui/package.json

```diff
@@ -18,12 +18,14 @@
     "@vaadin/vaadin-accordion": "^2.0.0-alpha5",
     "@vaadin/vaadin-button": "^3.0.0-alpha1",
     "@vaadin/vaadin-context-menu": "^5.0.0-alpha2",
+    "@vaadin/vaadin-dialog": "^2.5.2",
```

Comment:
It's safe to go for P3-based v3 alpha https://github.com/vaadin/vaadin-dialog/releases

Reply:
Sounds good